pkg/podmounter/mppod/creator.go

```diff
@@ -126,7 +127,10 @@ func (c *Creator) Create(pod *corev1.Pod, pv *corev1.PersistentVolume) *corev1.P
 				{
 					Name: CommunicationDirName,
 					VolumeSource: corev1.VolumeSource{
-						EmptyDir: &corev1.EmptyDirVolumeSource{},
+						EmptyDir: &corev1.EmptyDirVolumeSource{
+							Medium:    corev1.StorageMediumMemory,
+							SizeLimit: resource.NewQuantity(10*1024*1024, resource.BinarySI),
+						},
```

Comment:
Might be worth migrating this to a constant (10MB?)

Reply:
Fixed in https://github.com/awslabs/mountpoint-s3-csi-driver/pull/438/commits/287fd35e948f0ec04f7323c5dee2da5dfa0592ed